bootstrap/components/sources/kafka-source-topology-component.json

```diff
@@ -37,10 +37,11 @@
       {
         "uiName": "Reader schema version",
         "fieldName": "readerSchemaVersion",
-        "isOptional": true,
+        "isOptional": false,
```

Comment:
@priyank5485 I understand component spec jsons are not exported but the fields mentioned in uispec are represented by respective components which are stored in the backend storage. Earlier topology config's  kafka source will not have `readerSchemaVersion` in their config and when UI tries to load that in editor it may recognize that a mandatory field is missing and it may be throwing validation error. 

@shahsank3t @priyank5485 Pl let me know if I am missing anything here. I am just cautious whether we are breaking any topologies saved in earlier versions before this PR change.